app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -90,6 +100,17 @@ def mfa_verify_phone_otp_sent(reauthentication:, phone_number:, success:)
       )
     end
 
+    # @param [Boolean] success - True if the sms otp submitted matched what was sent
+    # During a login attempt, the user, having previously been sent an OTP code via SMS
+    # has entered an OTP code.
+    def mfa_verify_phone_otp_submitted(reauthentication:, success:)
+      track_event(
+        :mfa_verify_phone_otp_submitted,
+        reauthentication: reauthentication,
```

Comment:
should/can we document the reauthentication param?

Reply:
I was going to add this to the [schema doc](https://docs.google.com/document/d/1H7SAM8DSnbRJqKoSj4Qdzr-UvLCNMupt2zoqHLsIFe8/edit#heading=h.kz4mwmc93y6y). Is that sufficient?

Reply:
yes but I specifically meant the YARD-style documentation above the method. we're not making a hard requirement of it for this file so it may not matter

Reply:
actually scratch that, as of #6730, documentation is required of that param.

If you run lints, you'll get to:

```
$ make lint_tracker_events
# ...
app/services/irs_attempts_api/tracker_events.rb:106 mfa_verify_phone_otp_submitted reauthentication (undocumented)
make: *** [lint_tracker_events] Error 1

```